Jenkinsfile

```diff
@@ -78,6 +101,23 @@ pipeline {
 		}
 	}
 	post {
+		success {
```

Comment:
@Nazgolze I'm wondering if we could simply use `changed` here:
```
post {
  [...]
  changed {
    script {
      slack_send()
    }
  }
}
```

Reply:
@fchavant We can create a separate issue for that change if needed. I would like to keep this issue for applying the change made for the 2.0.0 only.